homeassistant/components/anglian_water/__init__.py

```diff
@@ -0,0 +1,72 @@
+"""Anglian Water (UK) integration."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from pyanglianwater import API, AnglianWater
+from pyanglianwater.exceptions import ServiceUnavailableError
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import CONF_DEVICE_ID, DOMAIN
+from .coordinator import AnglianWaterDataUpdateCoordinator
+
+type AnglianWaterConfigEntry = ConfigEntry[AnglianWaterConfig]
+
+PLATFORMS: list[Platform] = [
+    Platform.SENSOR,
+]
+
+
+@dataclass
+class AnglianWaterConfig:
+    """Represent a config for Anglian Water."""
+
+    coordinator: AnglianWaterDataUpdateCoordinator
```

Comment:
If you're only storing a coordinator, just only keep the coordinator in there instead of wrapping it in a class